src/lib.rs

```diff
@@ -1327,6 +1396,16 @@ fn blocks_required<T>(num_bits: usize) -> usize {
     num_bits / bits_per_block::<T>() + usize::from(num_bits % bits_per_block::<T>() != 0)
 }
 
+/// Convenience function that calls T::count_ones() and converts the result to usize
+/// (The conversion is always safe even if T is u128 and usize is u16)
+#[inline(always)]
+fn count_ones_usize<T: PrimInt>(value: T) -> usize {
```

Comment:
I'd prefer to call the `count_ones` std lib method directly rather than use this wrapper. The `try_from` is mildly ugly in source code, but it'll be optimised away, so it's not worth worrying about.